client/client-multi/client-objective-c/Kaa/KaaClientFactory.m

```diff
@@ -23,12 +23,16 @@
 @implementation KaaClientFactory
 
 + (id<KaaClient>)clientWithContext:(id<KaaClientPlatformContext>)context stateDelegate:(id<KaaClientStateDelegate>)delegate {
+    return [self clientWithContext:context stateDelegate:delegate autoGeneratedKeys:NO];
```

Comment:
I'd make the default strategy to generate keys for the user but not expect them to be already generated.